package.json

```
@@ -127,7 +127,7 @@
     ]
   },
   "engines": {
-    "node": ">=18 <21"
+    "node": "^18 || ^19 || ^20 || ^22"
```

I don't believe this is a valid syntax, let's try:

```suggestion
    "node": "18 || 19 || 20 || 22"
```